kolibri_explore_plugin/assets/src/searchApi.js

```diff
@@ -0,0 +1,56 @@
+import { KolibriApi } from './kolibriApi';
+
+function zimSearch(zimfile, query, node, maxResults = 10, start) {
```

Comment:
And if `node` can be removed from the parameters, it would be better to pass query, maxResults, start in a `searchParams` object directly.

Reply:
The `node` is the `ContentNode` with the zim file attached, that way we can redirect to the real node and add the node information in the `zimToNode` function.

This is done that way because a `cotentNode` can have more than one zim file, so a `zimSearch` promise is created for each zim file in each ContentNode, so after all promises, the node could be different. But maybe this could be made with more than one group of promises.